azavea/temperate
# Update help button copy and change plan profile capitalization

## Overview
Makes two copy changes:
- Updates reference to the help button to reflect its new label
- Fixes capitalization for the plan assessment

### Demo
<img width="484" alt="screen shot 2018-04-03 at 10 41 20 am" src="https://user-images.githubusercontent.com/1032849/38256210-c0720c6a-372b-11e8-9869-4cc0236d24d5.png">
<img width="463" alt="screen shot 2018-04-03 at 10 44 20 am" src="https://user-images.githubusercontent.com/1032849/38256333-fca92d1c-372b-11e8-8871-339516587a32.png">

## Testing Instructions
- In the admin panel, edit your user's organization, uncheck the "Plan setup complete" option, and save
- Open the Angular site
  - You should see the plan setup introduction with the updated copy
- Finish the wizard, advance to the dashboard, and click "Manage" under "City Profile"
- Open the "Assessments" drawer
  - The "I don't know" button should have sentence-style capitalization


Closes #1052
